.husky/commit-msg

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
```

Comment:
Could your revert this file and `.husky/pre-commit`

Reply:
I've removed the `readlink` now, or do you want me to revert everything? If yes, why?